### ISSUE TITLE
Странный визуальный баг в некоторых проектах

### ISSUE THREAD

**Issue by khokm** [khokm/stratum-player#35]
Непонятный квадратик при старте в некоторых проектах. Появляется не всегда.
![image](https://user-images.githubusercontent.com/25861783/128914045-a61388cd-ded3-4e07-bb2d-afcdcf6ad39d.png)

**Comment by Salaron** [comment]
Тултип без текста?
![image](https://user-images.githubusercontent.com/27885055/128927840-0ebab2dd-185c-43ec-8941-686fa5efe888.png)

**Comment by khokm** [comment]
@Salaron да, он самый. Решение проблемы довольно простое: @Sogi-IIT, нужно заменить здесь `SW_SHOW` на `SW_HIDE`.
![2021-08-11_01-24-28](https://user-images.githubusercontent.com/25861783/128930885-31d55a7e-627e-4afb-8775-e9215fafad6e.png)

**Comment by Sogi-IIT** [comment]
ага зафиксировал. решение рабочее. можно закрывать
